codalab/rest/interpret.py

```diff
@@ -347,7 +347,7 @@ def fetch_interpreted_worksheet(uuid):
     # Frontend doesn't use individual 'items' for now
     del worksheet_info['items']
     if bundle_uuids:
-        return {'blocks': worksheet_info['blocks']}
+        return {'blocks': worksheet_info['blocks'], 'uuid': uuid}
```

Comment:
What is `uuid` equal to? I thought `bundle_uuids` can specify multiple bundle uuids?

Reply:
`uuid` refers to the worksheet's uuid. `bundle_uuids` refers to the bundles' uuids.